src/Ocelot.Provider.Polly/PollyResiliencePipelineDelegatingHandler.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Ocelot.Configuration;
+using Ocelot.Logging;
+using Ocelot.Provider.Polly.Interfaces;
+using Polly.CircuitBreaker;
+using System.Diagnostics;
+
+namespace Ocelot.Provider.Polly;
+
+public class PollyResiliencePipelineDelegatingHandler : DelegatingHandler
+{
+    private readonly DownstreamRoute _route;
+    private readonly IHttpContextAccessor _contextAccessor;
+    private readonly IOcelotLogger _logger;
+
+    public PollyResiliencePipelineDelegatingHandler(
+        DownstreamRoute route,
+        IHttpContextAccessor contextAccessor,
+        IOcelotLoggerFactory loggerFactory)
+    {
+        this._route = route;
```

Comment:
remove the this pointers, no? Probably, Resharper got mad...

Reply:
That's why Microsoft C# coding conventions recommend to use `_` as the first character for private members of the class.
But this will be OK 👉  `this.route = route;` if defined as `object route;`

Reply:
Fixed in https://github.com/ThreeMammals/Ocelot/pull/1914/commits/f4aca4e0b749e5d6d185aa1e4d7866cde3561fc7